content/static/js/custom.cfr.js

```diff
@@ -17,4 +20,64 @@ function checkNavbar() {
   } else {
     showNavbar();
   }
-};
\ No newline at end of file
+};
+
```

Comment:
[NON-ESSENTIAL] Because this code is specific to the cfr page, it should only be included on the cfr page (either in a separate .js file or embedded into the cfr.html file with a script tag).

Reply:
Done.